src/databricks/labs/remorph/connections/database_manager.py

```diff
@@ -4,6 +4,7 @@
 
 from sqlalchemy import create_engine
 from sqlalchemy.engine import Engine, Result, URL
+from snowflake.sqlalchemy import URL as SnowflakeURL
```

Comment:
I always go back and forth on this, using Snowflake Connector Lib vs. using vanilla pyodbc with Alchemy. 
lesser library management and other advantages.

Reply:
Should I pull this out of this PR for now? I only added it since I was adding a Postgres connector anyway.

Reply:
Leave it as is, we will add it in Notice in a separate PR.